mava/components/jax/executing/base.py

```diff
@@ -46,7 +47,18 @@ def on_execution_init_start(self, executor: SystemExecutor) -> None:
         Returns:
             None.
         """
-        executor._interval = self.config.interval  # type: ignore
+
+        if self.config.evaluation_duration != 1 and (
+            self.config.evaluation_interval is None
+        ):
+            raise ValueError(
+                "Missing evaluation interval value: Evaluation duration"
+                + " value was provided without a value for the evaluation interval."
+            )
+
+        if executor.store.is_evaluator:
+            executor.store.evaluation_interval = self.config.evaluation_interval  # type: ignore # noqa: E501
+            executor.store.evaluation_duration = self.config.evaluation_duration  # type: ignore # noqa: E501
```

Comment:
think this variable should be explicitly called evaluation_episode_duration

Reply:
I think it is nice to leave it as is for now so that we have flexibility to change it to a dictionary as well in the future? Then we could evaluate over number of steps or even for a fixed amount of time. What are your thoughts?

Reply:
I guess that is fair but until we change it to a dict, it can be confusing for new people to the code

Reply:
Or maybe `num_episodes_per_eval`? Otherwise, we can change it to a dict now. Making it more specific is probably a good idea. But I get what both of you are saying. Maybe we can create a dictionary of one entry in that case? @RuanJohn @OmaymaMahjoub what do you think?

Reply:
Done here are [the changes ](https://github.com/instadeepai/Mava/pull/709/commits/1943dcb7569bbfa4bf8c27df3fc2362176d36244)!